### ISSUE TITLE
docs: support github markdown

### ISSUE THREAD

**Issue by kiwicopple** [supabase/dbdev#162]
# Chore

## Describe the chore

dbdev deoesn't support all of gh flavored markdown:

<img width="818" alt="image" src="https://github.com/supabase/dbdev/assets/10214025/353745ee-4fc4-4a0e-a8d1-6a74caf10925">

## Additional context

https://database.dev/kiwicopple/supa_audit

from this extension:

https://github.com/supabase/supa_audit

**Comment by alaister** [comment]
Yes, I think `skipHtml` seems to be only for trusted environments: https://github.com/remarkjs/react-markdown?tab=readme-ov-file#appendix-a-html-in-markdown

I'll investigate what the proper method is here.
